### PR TITLE
build_model refs replaced

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 The main purpose of this code is to demonstrate how to upload the model to the platform, optimize and download it
  after training is complete, using DeciPlatformCallback.
 """
+from charset_normalizer import models
```
Comment (ofrimasad) [inline src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py:7]: import **what?**

File: src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py [hunk 2]
```diff
@@ -52,4 +51,7 @@
                                                                             last_img_idx_in_batch=4)],
                 }
 
-trainer.train(train_params)
+trainer.train(training_params=train_params,
```
Comment (ofrimasad) [inline src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py:54]: please put the model first in the params (just to keep it clean)

File: src/super_gradients/training/utils/quantization_utils.py [hunk 3]
```diff
@@ -261,31 +262,21 @@ def _validate_args(self, start_epoch: int, quant_modules_calib_method: str, cali
 
     def __call__(self, context: PhaseContext):
         if context.epoch == self.start_epoch:
-            # SET CHECKPOINT PARAMS SO WE LOAD THE BEST CHECKPOINT SO FAR
```
Comment (ofrimasad) [inline src/super_gradients/training/utils/quantization_utils.py:264]: why was that removed?

**Review by ofrimasad — COMMENTED** [review]
my comments inline

**Review by ofrimasad — CHANGES_REQUESTED** [review]
please remove sg_model and kd_model libraries as well

File: src/super_gradients/training/utils/quantization_utils.py (hunk 3 above)
Comment (shaydeci) [reply src/super_gradients/training/utils/quantization_utils.py:264]: Checkpoint_params aren't used anymore since build_model is removed (when using get, with a SgTrainer checkpoint it will automatically take ema net. Other checkpoint params aren't relevant here either..bottom line its the same behaviour)

File: src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/examples/deci_lab_export_example/deci_lab_export_example.py]: fixed in https://github.com/Deci-AI/super-gradients/pull/364/commits/b1b57b1e73601102855d9e3b9c31691489cb98b2

File: src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py (hunk 2 above)
Comment (shaydeci) [reply src/super_gradients/examples/regseg_transfer_learning_example/regseg_transfer_learning_example.py]: fixed in https://github.com/Deci-AI/super-gradients/pull/364/commits/b1b57b1e73601102855d9e3b9c31691489cb98b2

**Review by ofrimasad — APPROVED** [review]
LGTM
